ubtue_Brill.js

```diff
@@ -153,10 +153,17 @@ function postProcess(doc, item) {
 	// mark articles as "LF" (MARC=856 |z|kostenfrei), that are published as open access
 	let openAccessTag = text(doc, '.has-license span');
 	if (openAccessTag && openAccessTag.match(/open\s+access/gi)) item.notes.push({note: 'LF:'});
```

Comment:
as discussed: 
To avoid multiple pushes of `LF` to `item.notes`:
```js
let is_LF = false;
...
//replace item.notes.push('LF:') by is_LF = true
...
//After all tests 
if (is_LF)
    item.notes.push('LF:');
```
For performance reasons the previous and new case can be written e.g.
```js
if (freeAccess && freeAccess.match(/(free|freier)\s+(access|zugang)/gi))
   is_LF = true;
else {
    let scriptItems = doc.querySelectorAll('head > script');
    for (let i of scriptItems) {
          if (i.text.includes('free-public')) {
               is_LF = true;
               break;
          }
     }
}
```

Remark: The new code has to search through all head scripts and is thus slow. I still doubt the benefit considering how many Brill journals we have.